Ira/my-app-video/src/redux/movies/reducer.ts

```diff
@@ -0,0 +1,67 @@
+import {IFilm} from "../../interfaces/IFilm";
+import {MovieActionTypes, moviesAction} from "./actionTypes";
+
+export interface IMovieInitialState {
+  movies: IFilm[],
+  loading: boolean,
+  currentMovie: IFilm | null,
+  totalMovies: number,
+}
+
+const initialState: IMovieInitialState = {
+  movies: [],
+  loading: false,
+  currentMovie: null,
+  totalMovies: 0,
+};
+
+export const MovieReducer = (state = initialState, action: moviesAction) => {
```

Comment:
optional: 
better to rename to Movie**s**Reducer, because you keep list of movies here.